examples/todoapp/src/App.tsx

```diff
@@ -32,10 +33,28 @@ function App() {
             allTodos
               @fabrixView(
                 input: [
-                  { field: "collection", config: { label: "Your tasks" } }
+                  { field: "collection", config: { label: "タスク一覧" } }
+                  { field: "collection.id", config: { hidden: true } }
+                  { field: "collection.hasDone", config: { label: "Status" } }
+                  {
+                    field: "collection.actions"
+                    config: {
+                      label: "操作"
+                      index: -1
```

Comment:
Does `-1` mean before the id with index `0`?

Reply:
Yes, indexes are automatically assigned to all fields and it starts from zero, so giving `-1` means moving the field to  the most left.